### PR TITLE
doc: Update doc/bips.md with recent changes in master

### PR DESCRIPTION
**Pull request by maflcko**
Follow-up to #17165

### CODE REVIEW

File: doc/bips.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.19.0**):
...
+* [`BIP 70`](https://github.com/bitcoin/bips/blob/master/bip-0070.mediawiki) [`71`](https://github.com/bitcoin/bips/blob/master/bip-0071.mediawiki) [`72`](https://github.com/bitcoin/bips/blob/master/bip-0072.mediawiki): Payment Protocol support has been available in Bitcoin Core GUI since **v0.9.0** ([PR #5216](https://github.com/bitcoin/bitcoin/pull/5216)). Support can be optionally disabled at build time since **v0.18.0** ([PR 14451](https://github.com/bitcoin/bitcoin/pull/14451)), and is disabled by default at build time since **v0.19.0** ([PR #15584](https://github.com/bitcoin/bitcoin/pull/15584)). It has been completely removed as of **v0.20.0** ([PR 17165](https://github.com/bitcoin/bitcoin/pull/17165)).
```
Comment (jonatack) [inline doc/bips.md]: Perhaps replace "It has been completely removed" with "It has been removed" (no need for "completely") or "Payment Protocol support has been removed".

Given the two sentences before use the present tense I was going to suggest the same ("Support is removed"), but saw that the verb tenses vary in this document between present tense and various forms of past tense.

Comment (jonatack) [reply doc/bips.md]: nit: Unrelated, but "14451)), and" should be "14451)) and" (no comma) or "14451)), and it" (comma followed by an independant clause)

**Review by jonatack — COMMENTED** [review]
ACK with a couple nits below.

**Comment by emilengler** [comment]
ACK

**Comment by jonatack** [comment]
ACK fa7f5a4d2a2581cc25125311892a80efc2c494e2. Verified markdown view at https://github.com/MarcoFalke/bitcoin-core/blob/1911-docBips/doc/bips.md and the urls in the links. Some of the PRs are indicated with # and some without, but this is the case over the whole document.

**Comment by laanwj** [comment]
ACK fa7f5a4d2a2581cc25125311892a80efc2c494e2

**Review by fanquake — APPROVED** [review]
ACK fa7f5a4d2a2581cc25125311892a80efc2c494e2

**Pull request merged**